components/eventing-controller/pkg/backend/jetstreamv2/test_helpers.go

```diff
@@ -24,9 +24,10 @@ import (
 )
 
 const (
-	DefaultStreamName    = "kyma"
-	DefaultMaxReconnects = 10
-	DefaultMaxInFlights  = 10
+	DefaultStreamName             = "kyma"
+	DefaultJetStreamSubjectPrefix = "kyma"
```

Comment:
(nit) this will not clash with any other test right?

Reply:
nope, this is just a default prefix for stream name, so in the tests I am using like:
```
fmt.sprintf("%s%d", DefaultStreamName, natsPort)
```